components/app-proxy/Bootstrap-Application-Proxy.ps1

```diff
@@ -43,10 +43,10 @@ $AppProxyFolder = "C:\Program Files\Microsoft AAD App Proxy Connector"
 
 $attempts = 0
 # it takes a bit of time for the application proxy folder to get created
-while (-not(Test-Path -Path $AppProxyFolder)) {
+while (-not(Test-Path -Path $AppProxyFolder\Modules)) {
```

Comment:
Do we not want to fix the reinstall attempt that caused this issue initially?

Reply:
correcting the path should mitigate the issue we saw last time